modules/apps/foundation/user-associated-data/user-associated-data-impl/src/main/java/com/liferay/user/associated/data/internal/registry/UADRegistryImpl.java

```diff
@@ -155,6 +227,30 @@ public void map(
 			String uadEntityClassName = (String)serviceReference.getProperty(
 				"model.class.name");
 
+			System.out.println(
+				"############## Register anonymizer : " + uadEntityClassName);
+
+			if (Validator.isNotNull(uadEntityClassName)) {
+				emitter.emit(uadEntityClassName);
+			}
+		}
+
+	}
+
+	private class UADEntityDisplayReferenceMapper
+		implements ServiceReferenceMapper<String, UADEntityDisplay> {
+
+		@Override
+		public void map(
+			ServiceReference<UADEntityDisplay> serviceReference,
+			Emitter<String> emitter) {
+
+			String uadEntityClassName = (String)serviceReference.getProperty(
+				"model.class.name");
+
+			System.out.println(
+				"############## Register display : " + uadEntityClassName);
```

Comment:
Are these system outs for dev purposes only? Or should we convert these to using a logger at the info level?